mvp.feature

```diff
@@ -1102,3 +1104,39 @@ Feature: MVP features for NVA
       | Institution DNS               |
       | Administration ID             |
       | Feide Organization ID         |
+
+  @902
+  Scenario: User opens a Public Profile from a Public Page
+    Given the user has opened NVA
+    And sees a Public Page
```

Comment:
Noe må skje her før de ser på en public profile. Hvor klikker de? Dette er viktig for å kunne skrive tester

Reply:
You could potentially use "Creator publishes Publication"